ui/src/app/pages/access/controlled-tier-card.tsx

```diff
@@ -63,15 +63,17 @@ const ControlledTierEraModule = (props: {
 const ControlledTierStep = (props: {
   enabled: boolean;
   text: String;
+  description: String;
   style?;
 }) => {
+  const { enabled, text, description, style } = props;
   return (
-    <FlexRow style={props.style}>
+    <FlexRow alt={description} style={style}>
```

Comment:
Do our `FlexRow`s support `alt` tags? The implementation uses a `div`, which [I don't think support alt tags](https://stackoverflow.com/a/62000730).

Reply:
The tests were passing, but FlexRows are divs when you get to their root, so I think it's best to change it. I changed it to make use of the title attribute instead of the alt attribute, and that seems to be working well. Thanks for pointing this out.